### DISCUSSION TITLE
DataGridAggregates As Top Row?

### DISCUSSION THREAD

**Discussion by cdodrzywolski** [Megabit/Blazorise#2306] [Q&A]
Hello,

I have a DataGrid with DataGridAggregates and it is working correctly. However the users would like to see the Aggregates displayed as the top row of the data instead of showing as the bottom row. Is there any setting/tricks to get the Aggregates to show on top instead of on the bottom?

**Comment by stsrki** [comment]
I'm not even sure where on top it could be placed. The top is meant to be used for the table header.

Can you provide a sample screenshot or some UI drawing how would it look like?

**Reply by cdodrzywolski** [reply]
Hmmm I see your point once I draw it out, It looks odd having the totals be above the column names, and if I put it below then it seems odd as its a data row. 

Maybe I can make a fake data row for Totals or display it in a section above or something.

![image](https://user-images.githubusercontent.com/68967069/117325336-9c74ef00-ae5e-11eb-858f-22e23ad48a6c.png)

**Reply by stsrki** [reply]
It makes more sense to have them above the table(DataGrid) as a separate part of UI.

**Comment by cdodrzywolski** [comment]
After looking at this in practice I see strski's point better off to just have these abobe the table in another DataGrid or some other part of the UI.
